superset-frontend/src/SqlLab/actions/sqlLab.js

```diff
@@ -1419,7 +1419,8 @@ export function popDatasourceQuery(datasourceKey, sql) {
       .then(({ json }) =>
         dispatch(
           addQueryEditor({
-            name: `Query ${json.result.name}`,
+            // eslint-disable-next-line prefer-template
+            title: t('Query') + ` ${json.result.name}`,
```

Comment:
Same here, might as well lean into the template strings unless there's a reason not to. 
```suggestion
            title: `${t('Query')} ${json.result.name}`,
```